howto/android/odm_vhal.md

```diff
@@ -0,0 +1,137 @@
+*since 1.22.0*
+
+[note type="information" status="Note"]Replacing the Anbox VHAL is only supported on AAOS images.
```

Comment:
At best we link to the image overview here to allow people identify these easily.